draft-ietf-rats-corim.md

```diff
@@ -456,9 +490,26 @@ The following describes each child item of this group.
 ~~~
 
 
-# CoMID {#sec-comid}
+# Concise Module Indentifier (CoMID) {#sec-comid}
+
+A CoMID tag contains information about hardware or firmware component or module to provision
```

Comment:
change: "hardware or firmware" to "hardware, firmware, or module composition that is provisioned"